src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +46,11 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Movie> getAllMovies = session.createQuery("from Movie ");
```

Comment:
It is better to write all SQL commands in upper case
```suggestion
            Query<Movie> getAllMovies = session.createQuery("FROM Movie ");
```